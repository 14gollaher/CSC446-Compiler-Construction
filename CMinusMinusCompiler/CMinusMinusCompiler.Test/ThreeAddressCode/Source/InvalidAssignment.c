int main()
{
	const z = 2.5;
	z = z;

	return -1;
}