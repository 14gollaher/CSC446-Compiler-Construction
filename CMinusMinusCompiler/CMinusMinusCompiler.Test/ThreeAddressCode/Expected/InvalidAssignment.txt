Proc main
	_bp-2 = 2.5
ERROR: Line 5 Invalid assignment to z
Start Proc Main